docs/exchanges/ascend-ex.md

```diff
@@ -8,16 +8,20 @@ tags:
 
 ## 📁 [Connector folder](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/ascend_ex)
 
+## 📝 Signup
+
+* Hummingbot referral link - <https://ascendex.com/register?inviteCode=UEIXNXKW>
+* Sign up with AscendEX using Hummingbot's referral link for a 10% discount!
```

Comment:
Add exchange-specific details here, from https://www.notion.so/hummingbot-foundation/AscendEx-46bf7d175e1c41e6b876f7ff4794f713

* Sign up with AscendEX using Hummingbot's referral link for a 10% discount off trading fees, for 365 days after registration